+// Top-level build file where you can add configuration options common to all sub-projects/modules.
diff --git a/node_modules/react-native-vector-icons/fonts.gradle b/node_modules/react-native-vector-icons/fonts.gradle
index 490756c..7a038c0 100644
--- a/node_modules/react-native-vector-icons/fonts.gradle
+++ b/node_modules/react-native-vector-icons/fonts.gradle
@@ -20,22 +20,24 @@ gradle.projectsEvaluated {
 
             iconFontNames.each { fontName ->
 
-              from(iconFontsDir) {
-                include(fontName)
-                into("assets/${targetPath}/fonts/")
+              copy {
+                from fileTree(iconFontsDir).include(fontName).files
+                into("src/main/assets/fonts/")
               }
 
+              /*
               // Workaround for Android Gradle Plugin 3.2+ new asset directory
-              from(iconFontsDir) {
-                include(fontName)
-                into("merged_assets/${variant.name}/merge${targetName}Assets/out/fonts/")
+              copy {
+                from fileTree(iconFontsDir).include(fontName).files
+                into("src/main/merged_assets/${variant.name}/merge${targetName}Assets/out/fonts/")
               }
 
               // Workaround for Android Gradle Plugin 3.4+ new asset directory
-              from(iconFontsDir) {
-                include(fontName)
-                into("merged_assets/${variant.name}/out/fonts/")
+              copy {
+                from fileTree(iconFontsDir).include(fontName).files
+                into("src/main/merged_assets/${variant.name}/out/fonts/")
               }
+              */
             }
         }
 
diff --git a/node_modules/react-native-vector-icons/lib/create-icon-set.js b/node_modules/react-native-vector-icons/lib/create-icon-set.js
index 93a7418..ef1df18 100644
--- a/node_modules/react-native-vector-icons/lib/create-icon-set.js
+++ b/node_modules/react-native-vector-icons/lib/create-icon-set.js
@@ -11,7 +11,7 @@ import {
 import ensureNativeModuleAvailable from './ensure-native-module-available';
 import createIconButtonComponent from './icon-button';
 import createTabBarItemIOSComponent from './tab-bar-item-ios';
-import createToolbarAndroidComponent from './toolbar-android';
+// import createToolbarAndroidComponent from './toolbar-android';
 
 export const NativeIconAPI =
   NativeModules.RNVectorIconsManager || NativeModules.RNVectorIconsModule;
@@ -174,10 +174,10 @@ export default function createIconSet(
     getImageSource
   );
   Icon.TabBarItemIOS = Icon.TabBarItem;
-  Icon.ToolbarAndroid = createToolbarAndroidComponent(
-    IconNamePropType,
-    getImageSource
-  );
+  // Icon.ToolbarAndroid = createToolbarAndroidComponent(
+  //   IconNamePropType,
+  //   getImageSource
+  // );
   Icon.getImageSource = getImageSource;
   Icon.loadFont = loadFont;
   Icon.hasIcon = hasIcon;
